openpype/modules/deadline/plugins/publish/submit_publish_job.py

```diff
@@ -325,9 +329,18 @@ def _submit_deadline_post_job(self, instance, job, instances):
 
         return deadline_publish_job_id
 
-
     def process(self, instance):
         # type: (pyblish.api.Instance) -> None
+
+        r''' SUBMiT PUBLiSH JOB 2 D34DLiN3
+          ____
+        '     '            .---.  .---. .--. .---. .--..--..--..--. .---.
+        |     |   --= \   |  .  \/   _|/    \|  .  \  ||  ||   \  |/   _|
+        | JOB |   --= /   |  |  ||  __|  ..  |  |  |  |;_ ||  \   ||  __|
+        |     |           |____./ \.__|._||_.|___./|_____|||__|\__|\.___|
+        ._____.
+
+        '''
```

Comment:
```suggestion
```
This is way too high - it's not close to whether the actual job gets submitted whereas previously it was. This should be removed or moved back to the older position in the code.